internal/setup/setup_test.go

```diff
@@ -0,0 +1,201 @@
+// Copyright 2020 Google LLC
```

Comment:
yeah - maybe just submit this, and I think it closes out the "things that could go wrong" category.

The mains are as simple as they can be currently.

Reply:
Done